ansible/roles/devspace/tasks/devspace-install.yml

```diff
@@ -119,7 +120,6 @@
     regexp: '^ARG USER_ID=1000'
     replace: 'ARG USER_ID={{ devspace_user_id }}' # how to find out UID?
   with_items:
-    - bf/Dockerfile
```

Comment:
Should I close https://github.com/openmicroscopy/infrastructure/pull/235 in favor of this PR?